docs/appendix/yaml.rst

```diff
@@ -126,6 +136,65 @@ data (e.g., surf_only: True).
 
+**extra_calc:** The extra_calc section allows users to calculate complex meteorological 
+variables that may not have standard variable names across datasets. E.g. u, v, u10, and v10
+all commonly refer to the u and v components of wind speed. However, the naming convention
+varies by model. 
+
+This section allows dewpoint, relative humidity, wind speed, and wind direction to be 
+calculated. Users can also specify whether their spatial overlay, spatial bias, and spatial 
+exceedance plots display wind barbs. 
+
+**NOTE:** All extra_calc calculations should be left in SI units. See example yaml files:
+   
+   * ``control_ish_lite_ufschem-example.yaml``
+   * ``control_ish_ufschem-example.yaml``
+   * ``control_ufschem-example.yaml``
+
+* **Hydrometeorology:** 
+
+   * Observations will sometimes only record dewpoint or relative humidity. Hence, both options are available for
+     calculation, provided the model has specific humidity. 
+
+   * **Model dewpoint:** can be calculated with the specific humidity. Users will need to know the
+     naming convention of specific humidity in their specific model. Observations will sometimes 
+     only record dewpoint or relative humidity. Hence, both options are available for calculation,
+     provided the model has specific humidity. 
+ 
+   * **Model relative humidity:** can be calculated with the specific humidity. Users will need to know 
+     the naming convention of specific humidity in their specific model. 
+
+   * *Calculations for standardized hydrometeorological variables continue to be incorporated into MELODIES-MONET.*
+
+* **Wind speed and direction:**
+
+   * **Wind speed:** is calculated using the u-component and v-component. Users are responsible for
+     knowing what those components are called in their model. If modeled wind speed is already 
+     available, simply add the variable name to the variable list. 
+
+   * **Wind direction:** is calculated using the u-component and v-component. Users are responsible for
+     knowing what those components are called in their model. If modeled wind direction is already 
+     available, simply add the variable name to the variable list. 
+
+   * **Wind barbs:** can be plotted using the u-component and v-component. Users are responsible for
+     knowing what those components are called in their model. **NOTE: plotted wind barbs are in knots. 
+     Wind speed everywhere else in the model/observations are by default m/s unless specified elsewhere in the 
+     YAML options.** 
+
+* **Potential temperature:**
+
+   * **Modeled potential temperature:** is calculated using the modeled pressure and temperature. Users are responsible for
+     knowing what those components are called in their model. 
+
+   * **Observed potential temperature:** is calculated using the observed pressure and temperature. Users are responsible for
+     knowing what those components are called in their model.
+
```

Comment:
Can you update the yaml description to copy paste these items that can be calculated for the observations into the "Observations" section of this yaml description document. Then the "Observed potential temperature" would be moved to that section. This would make it clearer with Pablo's comment above and consistent with where you specify the observed potential temperature in this yaml file example: https://github.com/NCAR/MELODIES-MONET/pull/379/files#diff-b7adecc1e6bf6b842d3ab7e59ab1a907cf7fc88c56707f7169be851c6d29623d